R/estimateY.R

```diff
@@ -71,12 +71,49 @@ estimateY <- function(relationship, regfunctions,
                       includeRegUnc = TRUE,
                       distribution = "normal",
                       rangeRestrict = FALSE,
-                      rangeY = c(-Inf, +Inf)){
+                      rangeY = c(-Inf, +Inf),
+                      imputeMissing = TRUE){
```

Comment:
Can you add a description for `imputeMissing` above? Than the checks should be passing again.